db/migrate/20210222102602_create_users_notification_settings.rb

```diff
@@ -0,0 +1,24 @@
+class CreateUsersNotificationSettings < ActiveRecord::Migration[6.0]
+  def change
+    create_table :users_notification_settings do |t|
```

Comment:
same thing about using `t.references` instead of `t.integer :user_id` and various null checks if needed

Reply:
✅